elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
```

Comment:
These comments are helpful for the template. If you're creating new elements you should probably remove them.